### PR TITLE
Update permissions for synthetics getting started

### PR DESCRIPTION
**Pull request by shahzad31**
Add read permissions for synthetics and uptime alerts indices !!

### CODE REVIEW

File: docs/en/observability/synthetics-get-started-ui.asciidoc [hunk 1]
```diff
@@ -31,7 +31,7 @@ To enable Monitor Management, you must be logged into {kib} as a user with
 the following privileges:
 
 * {ref}/security-privileges.html#privileges-list-cluster[Cluster privileges]: `monitor`, `read_ilm`, `read_pipeline`, and at least one of `manage_own_api_key`, `manage_security`, or `manage_api_key`
-* {ref}/security-privileges.html#privileges-list-indices[Indices privileges] for `synthetics-*`: `view_index_metadata`, `create_doc`, `auto_configure`
+* {ref}/security-privileges.html#privileges-list-indices[Indices privileges] for `synthetics-*`: `read`, `view_index_metadata`, `create_doc`, `auto_configure` and for `.alerts-observability.uptime.alerts-*`: `read`
```
Comment (dominiqueclarke) [inline docs/en/observability/synthetics-get-started-ui.asciidoc:34]: We should probably also add Kibana `Synthetics and Uptime` all permissions.

**Review by colleenmcginnis — APPROVED** [review]
👍  If you add more permissions per @dominiqueclarke's comment, you can verify it is formatted correctly using [this preview link](https://observability-docs_2645.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-get-started-ui.html#uptime-set-up-prereq) (it will take several minutes for the preview to build).

**Comment by paulb-elastic** [comment]
Been discussing / testing with others in the team and the updated permissions are not enough for Private Locations that fail with just these (missing Fleet permissions).  Would be good to add the additional permissions needed to the [Private Locations docs page](https://www.elastic.co/guide/en/observability/current/synthetics-private-location.html#monitor-via-private-agent)

**Review by dominiqueclarke — CHANGES_REQUESTED** [review]
I just want to point out that these original permissions were created with the express purpose of describing the permission set needed to _enable_ synthetics. That is, the permission set needed to generate an api key that can be used by the service.

That is why there is a miss of Kibana permissions on here, as well as Fleet permissions for private locations. Both are irrelevant for generating the api key needed for Synthetics. 

That being said, it is clear that these docs are confusing, I'm just wondering what the right route to go here is. An admin needed to enable Synthetics and generate the api key needs a higher permission set than the average user. 

I want to distinguish between admins and Synthetics users. I'm not sure what is the best, clearest way to do so. @colleenmcginnis 

I don't necessarily want to lump all the permissions needs to use Synthetics as a user here, because I don't wnat to make it seem like ordinary synthetics users need cluster advanced cluster permissions.

Admin (Needed to generate the synthetics service api key for public locations)
```
* {ref}/security-privileges.html#privileges-list-cluster[Cluster privileges]: `monitor`, `read_ilm`, `read_pipeline`, and at least one of `manage_own_api_key`, `manage_security`, or `manage_api_key`
* {ref}/security-privileges.html#privileges-list-indices[Indices privileges] for `synthetics-*`: `read`, `view_index_metadata`, `create_doc`, `auto_configure`
* Kibana privileges: Synthetics/Uptime ALL
```

Uptime write user using private locations
```
* {ref}/security-privileges.html#privileges-list-indices[Indices privileges] for `synthetics-*`: `read` and for `.alerts-observability.uptime.alerts-*`: `read`
* Kibana privileges: Uptime/Synthetics: All, Fleet: All, Integrations: All
```


Uptime write user NOT using private locations
```
* {ref}/security-privileges.html#privileges-list-indices[Indices privileges] for `synthetics-*`: `read` and for `.alerts-observability.uptime.alerts-*`: `read`
* Kibana privileges: Uptime/Synthetics
```

AND if users want to use project monitors, they'll need
```
* {ref}/security-privileges.html#privileges-list-cluster[Cluster privileges]: At least one of `manage_own_api_key`, `manage_security`, or `manage_api_key`
```

**Comment by paulb-elastic** [comment]
To add to this, https://github.com/elastic/observability-docs/issues/2707 has been raised for the _use_ of Synthetics (as opposed to enabling MM)

**Pull request closed without merging**